### ISSUE TITLE
印章识别中文乱码

### ISSUE THREAD

**Issue by mavisyyc** [PaddlePaddle/PaddleOCR#9417]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：python tools/infer_e2e.py -c /home/yyw/yuyuechun/ppocr/paddleocr/PaddleOCR/configs/e2e/e2e_r50_vd_pg.yml -o Global.pretrained_model=/home/yyw/yuyuechun/ppocr/paddleocr/PaddleOCR-release-2.6/models/PGNet/best_accuracy Global.infer_img="/home/yyw/yuyuechun/stampocr/企业微信截图_d05b4011-c4fd-4126-860b-46c46c64ca64.png"

- 完整报错/Complete Error Message：
[2023/03/14 12:49:28] ppocr INFO: Architecture : 
[2023/03/14 12:49:28] ppocr INFO:     Backbone : 
[2023/03/14 12:49:28] ppocr INFO:         layers : 50
[2023/03/14 12:49:28] ppocr INFO:         name : ResNet
[2023/03/14 12:49:28] ppocr INFO:     Head : 
[2023/03/14 12:49:28] ppocr INFO:         character_dict_path : ppocr/utils/ppocr_keys_v1.txt
[2023/03/14 12:49:28] ppocr INFO:         name : PGHead
[2023/03/14 12:49:28] ppocr INFO:     Neck : 
[2023/03/14 12:49:28] ppocr INFO:         name : PGFPN
[2023/03/14 12:49:28] ppocr INFO:     Transform : None
[2023/03/14 12:49:28] ppocr INFO:     algorithm : PGNet
[2023/03/14 12:49:28] ppocr INFO:     model_type : e2e
[2023/03/14 12:49:28] ppocr INFO: Eval : 
[2023/03/14 12:49:28] ppocr INFO:     dataset : 
[2023/03/14 12:49:28] ppocr INFO:         data_dir : ./train_data/total_text/test
[2023/03/14 12:49:28] ppocr INFO:         label_file_list : ['./train_data/total_text/test/test.txt']
[2023/03/14 12:49:28] ppocr INFO:         name : PGDataSet
[2023/03/14 12:49:28] ppocr INFO:         transforms : 
[2023/03/14 12:49:28] ppocr INFO:             DecodeImage : 
[2023/03/14 12:49:28] ppocr INFO:                 channel_first : False
[2023/03/14 12:49:28] ppocr INFO:                 img_mode : BGR
[2023/03/14 12:49:28] ppocr INFO:             E2ELabelEncodeTest : None
[2023/03/14 12:49:28] ppocr INFO:             E2EResizeForTest : 
[2023/03/14 12:49:28] ppocr INFO:                 max_side_len : 768
[2023/03/14 12:49:28] ppocr INFO:             NormalizeImage : 
[2023/03/14 12:49:28] ppocr INFO:                 mean : [0.485, 0.456, 0.406]
[2023/03/14 12:49:28] ppocr INFO:                 order : hwc
[2023/03/14 12:49:28] ppocr INFO:                 scale : 1./255.
[2023/03/14 12:49:28] ppocr INFO:                 std : [0.229, 0.224, 0.225]
[2023/03/14 12:49:28] ppocr INFO:             ToCHWImage : None
[2023/03/14 12:49:28] ppocr INFO:             KeepKeys : 
[2023/03/14 12:49:28] ppocr INFO:                 keep_keys : ['image', 'shape', 'polys', 'texts', 'ignore_tags', 'img_id']
[2023/03/14 12:49:28] ppocr INFO:     loader : 
[2023/03/14 12:49:28] ppocr INFO:         batch_size_per_card : 1
[2023/03/14 12:49:28] ppocr INFO:         drop_last : False
[2023/03/14 12:49:28] ppocr INFO:         num_workers : 2
[2023/03/14 12:49:28] ppocr INFO:         shuffle : False
[2023/03/14 12:49:28] ppocr INFO: Global : 
[2023/03/14 12:49:28] ppocr INFO:     cal_metric_during_train : False
[2023/03/14 12:49:28] ppocr INFO:     character_dict_path : ppocr/utils/ppocr_keys_v1.txt
[2023/03/14 12:49:28] ppocr INFO:     character_type : CN
[2023/03/14 12:49:28] ppocr INFO:     checkpoints : None
[2023/03/14 12:49:28] ppocr INFO:     distributed : False
[2023/03/14 12:49:28] ppocr INFO:     epoch_num : 600
[2023/03/14 12:49:28] ppocr INFO:     eval_batch_step : [0, 1000]
[2023/03/14 12:49:28] ppocr INFO:     infer_img : /home/yyw/yuyuechun/stampocr/企业微信截图_d05b4011-c4fd-4126-860b-46c46c64ca64.png
[2023/03/14 12:49:28] ppocr INFO:     infer_visual_type : CN
[2023/03/14 12:49:28] ppocr INFO:     log_smooth_window : 20
[2023/03/14 12:49:28] ppocr INFO:     max_text_length : 50
[2023/03/14 12:49:28] ppocr INFO:     max_text_nums : 30
[2023/03/14 12:49:28] ppocr INFO:     pretrained_model : /home/yyw/yuyuechun/ppocr/paddleocr/PaddleOCR-release-2.6/models/PGNet/best_accuracy
[2023/03/14 12:49:28] ppocr INFO:     print_batch_step : 10
[2023/03/14 12:49:28] ppocr INFO:     save_epoch_step : 10
[2023/03/14 12:49:28] ppocr INFO:     save_inference_dir : None
[2023/03/14 12:49:28] ppocr INFO:     save_model_dir : ./output/pgnet_r50_vd_totaltext/
[2023/03/14 12:49:28] ppocr INFO:     save_res_path : ./output/pgnet_r50_vd_totaltext/predicts_pgnet.txt
[2023/03/14 12:49:28] ppocr INFO:     tcl_len : 64
[2023/03/14 12:49:28] ppocr INFO:     use_gpu : True
[2023/03/14 12:49:28] ppocr INFO:     use_visualdl : False
[2023/03/14 12:49:28] ppocr INFO:     valid_set : totaltext
[2023/03/14 12:49:28] ppocr INFO: Loss : 
[2023/03/14 12:49:28] ppocr INFO:     max_text_length : 50
[2023/03/14 12:49:28] ppocr INFO:     max_text_nums : 30
[2023/03/14 12:49:28] ppocr INFO:     name : PGLoss
[2023/03/14 12:49:28] ppocr INFO:     pad_num : 36
[2023/03/14 12:49:28] ppocr INFO:     tcl_bs : 64
[2023/03/14 12:49:28] ppocr INFO: Metric : 
[2023/03/14 12:49:28] ppocr INFO:     character_dict_path : ppocr/utils/ic15_dict.txt
[2023/03/14 12:49:28] ppocr INFO:     gt_mat_dir : ./train_data/total_text/gt
[2023/03/14 12:49:28] ppocr INFO:     main_indicator : f_score_e2e
[2023/03/14 12:49:28] ppocr INFO:     mode : A
[2023/03/14 12:49:28] ppocr INFO:     name : E2EMetric
[2023/03/14 12:49:28] ppocr INFO: Optimizer : 
[2023/03/14 12:49:28] ppocr INFO:     beta1 : 0.9
[2023/03/14 12:49:28] ppocr INFO:     beta2 : 0.999
[2023/03/14 12:49:28] ppocr INFO:     lr : 
[2023/03/14 12:49:28] ppocr INFO:         learning_rate : 0.001
[2023/03/14 12:49:28] ppocr INFO:         name : Cosine
[2023/03/14 12:49:28] ppocr INFO:         warmup_epoch : 50
[2023/03/14 12:49:28] ppocr INFO:     name : Adam
[2023/03/14 12:49:28] ppocr INFO:     regularizer : 
[2023/03/14 12:49:28] ppocr INFO:         factor : 0.0001
[2023/03/14 12:49:28] ppocr INFO:         name : L2
[2023/03/14 12:49:28] ppocr INFO: PostProcess : 
[2023/03/14 12:49:28] ppocr INFO:     mode : fast
[2023/03/14 12:49:28] ppocr INFO:     name : PGPostProcess
[2023/03/14 12:49:28] ppocr INFO:     point_gather_mode : align
[2023/03/14 12:49:28] ppocr INFO:     score_thresh : 0.5
[2023/03/14 12:49:28] ppocr INFO: Train : 
[2023/03/14 12:49:28] ppocr INFO:     dataset : 
[2023/03/14 12:49:28] ppocr INFO:         data_dir : ./train_data/total_text/train
[2023/03/14 12:49:28] ppocr INFO:         label_file_list : ['./train_data/total_text/train/train.txt']
[2023/03/14 12:49:28] ppocr INFO:         name : PGDataSet
[2023/03/14 12:49:28] ppocr INFO:         ratio_list : [1.0]
[2023/03/14 12:49:28] ppocr INFO:         transforms : 
[2023/03/14 12:49:28] ppocr INFO:             DecodeImage : 
[2023/03/14 12:49:28] ppocr INFO:                 channel_first : False
[2023/03/14 12:49:28] ppocr INFO:                 img_mode : BGR
[2023/03/14 12:49:28] ppocr INFO:             E2ELabelEncodeTrain : None
[2023/03/14 12:49:28] ppocr INFO:             PGProcessTrain : 
[2023/03/14 12:49:28] ppocr INFO:                 batch_size : 14
[2023/03/14 12:49:28] ppocr INFO:                 max_text_size : 512
[2023/03/14 12:49:28] ppocr INFO:                 min_crop_size : 24
[2023/03/14 12:49:28] ppocr INFO:                 min_text_size : 4
[2023/03/14 12:49:28] ppocr INFO:                 point_gather_mode : align
[2023/03/14 12:49:28] ppocr INFO:                 use_random_crop : False
[2023/03/14 12:49:28] ppocr INFO:                 use_resize : True
[2023/03/14 12:49:28] ppocr INFO:             KeepKeys : 
[2023/03/14 12:49:28] ppocr INFO:                 keep_keys : ['images', 'tcl_maps', 'tcl_label_maps', 'border_maps', 'direction_maps', 'training_masks', 'label_list', 'pos_list', 'pos_mask']
[2023/03/14 12:49:28] ppocr INFO:     loader : 
[2023/03/14 12:49:28] ppocr INFO:         batch_size_per_card : 14
[2023/03/14 12:49:28] ppocr INFO:         drop_last : True
[2023/03/14 12:49:28] ppocr INFO:         num_workers : 16
[2023/03/14 12:49:28] ppocr INFO:         shuffle : True
[2023/03/14 12:49:28] ppocr INFO: profiler_options : None
[2023/03/14 12:49:28] ppocr INFO: train with paddle 2.4.1 and device Place(gpu:0)
W0314 12:49:28.447005 2852347 gpu_resources.cc:61] Please NOTE: device: 0, GPU Compute Capability: 8.6, Driver API Version: 11.7, Runtime API Version: 11.7
W0314 12:49:28.449815 2852347 gpu_resources.cc:91] device: 0, cuDNN Version: 8.6.
[2023/03/14 12:49:29] ppocr WARNING: The shape of model params head.conv3.weight [37, 256, 3, 3] not matched with loaded params head.conv3.weight [6624, 256, 3, 3] !
[2023/03/14 12:49:29] ppocr INFO: load pretrain successful from /home/yyw/yuyuechun/ppocr/paddleocr/PaddleOCR-release-2.6/models/PGNet/best_accuracy
[2023/03/14 12:49:29] ppocr INFO: infer_img: /home/yyw/yuyuechun/stampocr/企业微信截图_d05b4011-c4fd-4126-860b-46c46c64ca64.png
[2023/03/14 12:49:30] ppocr INFO: The e2e Image saved in ./output/pgnet_r50_vd_totaltext/e2e_results/企业微信截图_d05b4011-c4fd-4126-860b-46c46c64ca64.png
[2023/03/14 12:49:30] ppocr INFO: success!
![图片](https://user-images.githubusercontent.com/99957407/224896561-6288fcc8-4f4e-45a2-baff-10b6d1baaae4.png)
![图片](https://user-images.githubusercontent.com/99957407/224896638-d479f554-f1f4-4c2e-964b-0097ab62967b.png)
请问下是为啥，谢谢！

**Comment by mavisyyc** [comment]
![图片](https://user-images.githubusercontent.com/99957407/224918166-088278ba-7805-46c3-9b83-b2a337b80a0f.png)
能出来中文了，但这为啥准确率这么低，请问有哪些参数可以调整下提高准确率吗

**Comment by mavisyyc** [comment]
@andyjpaddle 求问大佬！盼回复！感谢！

**Comment by Tuannnxlf** [comment]
我也在做印章是别能加个QQ交流下吗？2893154355

**Comment by kkiskkk** [comment]
> * [2023/03/14 12:49:28] ppocr INFO:     character_dict_path : ppocr/utils/ic15_dict.txt [2023/03/14 12:49:28] ppocr 信息：character_dict_path : ppocr/utils/ic15_dict.txt
>   [2023/03/14 12:49:28] ppocr INFO:     gt_mat_dir : ./train_data/total_text/gt [2023/03/14 12:49:28] ppocr 信息：gt_mat_dir：./train_data/total_text/gt
>   [2023/03/14 12:49:28] ppocr INFO:     main_indicator : f_score_e2e

请问后续是如何解决乱码问题？
我目前遇到的是：

检测不准确  并且识别也不准
